CSS/style.css

```diff
@@ -830,6 +845,148 @@ hr {
     width: 73%;
   }
 
+  .languages-list li {
+    display: flex;
+    flex-direction: column;
+    width: 122px;
+    height: 120px;
+    text-align: center;
+    align-items: center;
+    justify-content: center;
+    margin-right: 20px;
+  }
+
+  .languages-list ul {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .languages-list > ul > li {
+    display: flex;
+
+    /* background: #f7f7f9; */
+    border-radius: 8px;
+    margin-bottom: 12px;
+    margin-top: 21px;
+    width: 120px;
+    height: 112px;
+  }
+
+  .languages-list p {
+    font-size: 17px;
+  }
+
+  .contact {
+    display: flex;
+    width: 100%;
+    height: 920px;
+    flex-direction: column;
+    background-image: url("../img/form.png");
+    background-repeat: no-repeat;
+    background-size: inherit;
+    background-position-x: right;
+    background-color: #6070ff;
+    border-top-left-radius: 90px;
+  }
```

Comment:
Kindly update the styling of your contact section to match the figma design better🙏

Your design:
<img width="509" alt="image" src="https://user-images.githubusercontent.com/37116322/230654886-1c95224f-9ac9-44c7-881f-7e3bc53e1c13.png">

Figma design:
<img width="531" alt="image" src="https://user-images.githubusercontent.com/37116322/230654943-6138b1d6-936a-4b78-bab3-2e7fe13a4590.png">

A few things to note:
- Please reduce the spacing between the main title, paragraph and form
- Please remove the border radius on the form inputs 
- Please ensure your form inputs have the same font family with the rest of the page